pkg/kube/ready.go

```diff
@@ -184,7 +193,9 @@ func (c *ReadyChecker) IsReady(ctx context.Context, v *resource.Info) (bool, err
 			return false, nil
 		}
 	case *corev1.ReplicationController, *extensionsv1beta1.ReplicaSet, *appsv1beta2.ReplicaSet, *appsv1.ReplicaSet:
-		ok, err = c.podsReadyForObject(ctx, v.Namespace, value)
+		return c.podsReadyForObject(ctx, v.Namespace, value)
+	default:
+		c.log("Status %s is ready: %s/%s", v.Mapping.GroupVersionKind.Kind, v.Namespace, v.Name)
```

Comment:
It doesn't look like helm supports sub loggers which would be better here, so wrap the status part so it doesn't read like part of the log message 
```suggestion
		c.log("[status] %s is ready: %s/%s", v.Mapping.GroupVersionKind.Kind, v.Namespace, v.Name)
```

Reply:
Wasn't trying to make it look like a sublogger - I just needed a word to key off of when parsing the logs and looking for the "Ready" message.